homeassistant/components/gree/const.py

```diff
@@ -0,0 +1,22 @@
+"""Constants for the Gree Climate integration."""
+
+DOMAIN = "gree"
+
+CONNECTION_NETWORK_MAC = "mac"
+
+DEFAULT_DISCOVERY = True
+DEFAULT_NAME = "Gree HVAC"
+DEFAULT_PORT = 7000
+
+FAN_MEDIUMLOW = "medium low"
+FAN_MEDIUMHIGH = "medium high"
+
+MIN_TEMP = 16
+MAX_TEMP = 30
+
+MAX_ERRORS = 2
```

Comment:
Perhaps a comment here would be helpful, something like:

`# Under normal conditions GREE units timeout every once in a while.  This is the max number of errors before the device is marked unavailable.`

Reply:
My preference is to keep this comment near the logic that acts on this value--as this is the more critical pathway, and probably the likely place a developer will encounter the need to know this. My understanding is that these constants are not like a configuration file, values don't need to be explained away. Note, that this comment already exists where the error count is checked.

Reply:
I'll resolve this for now, but if a reviewer from HA itself asks for the same change, then I will add the comment here.